Testing_Coverage_task.py

```diff
@@ -0,0 +1,41 @@
+#Task1
+def my_range(start=0, stop=None, step=1):
+
+    if stop is None:
+        stop = start
+        start = 0
+
+    if start < stop and step>0:
+        while start<stop:
+            yield start
+            start+=step
+    else:
+        while start>stop and step<0:
+            yield start
+            start+=step
+
+#Task2
+def cached(use_cache):
+    def use(func):
+        cache = {}
+
+        def a(*args, **kwargs):
+            nonlocal cache
+
+            if not cache.get(args):

```

Comment:
Conditions could be combined (via `or` operator), for avoiding duplication of code.
Coverage also would be improved:
![image](https://user-images.githubusercontent.com/6921551/120300407-91ce3e00-c2d4-11eb-9e4d-86442aa74b0a.png)